index.html

```diff
@@ -487,10 +505,13 @@ <h2>Credentials</h2>
 
       <p>
 A <a>credential</a> is set of one or more <a>claims</a> made by the
-same <a>entity</a>. It may include an identifier to uniquely identify the
-credential, as well as metadata that describes properties of the credential
-itself such as: the <a>issuer</a>, the expiry time, a representative image,
-etc. A <a>verifiable credential</a> is a set of claims and meta data that are
+same <a>entity</a>. It includes a holder generated almost-always unique identifier
```

Comment:
`holder-generated identifier that is almost always unique` ... still, this language is crufty.

Reply:
Changed the language.